### DISCUSSION TITLE
Verbose `-v` flag doesn't work with `pytest --forked -v`?

### DISCUSSION THREAD

**Discussion by sudhakarsingh27** [pytest-dev/pytest-forked#73] [Q&A] [answered]
I recently switched to `pytest --forked -v ...` command line but it looks like the verbose flag doesn't print anything to the output. Is it expected? If yes, is there another way to print stdout and stderr from the tests?

**Comment by nicoddemus** [comment, marked as answer]
`-v` is about verbosity of what pytest itself outputs, but you mention `print stdout and stderr from the tests`, in which case seems you mean the `-s` flag?

Unfortunately `-s` does not work because `execnet`, the underlying library used for communication between master and workers, does not support it.

**Reply by RonnyPfannschmidt** [reply]
The real issue is that forked is unrelated to execnet and running the runtestprotocol in a fork without back report

**Reply by nicoddemus** [reply]
Ahh right, sorry.

**Reply by sudhakarsingh27** [reply]
Ah right my bad, I meant the `-s` flag. 

I see. Btw, when the test fails, the error is reported (with stack trace). ~~Does that mean that `stderr` is being reported back but not `stdout`?~~ (I realized it's just a stack trace.) 

How difficult is it to add reporting with `--forked` option?

**Reply by RonnyPfannschmidt** [reply]
Horrendously, it's currently using stdio to transfer reports instead of the io

**Reply by sudhakarsingh27** [reply]
Okay, thanks!
